lib/supplejack_common/request.rb

```diff
@@ -91,7 +93,8 @@ def delay
     end
 
     def request_url
-      ::Retriable.retriable(tries: 5, base_interval: 1, multiplier: 2) do
+      # TODO Reset
+      # ::Retriable.retriable(tries: 5, base_interval: 1, multiplier: 2) do
```

Comment:
When will you reset?

Reply:
Lol that a note to myself before I pushed, I'll clean it up now